### DISCUSSION TITLE
New release date?

### DISCUSSION THREAD

**Discussion by developera** [FreeOpcUa/opcua-asyncio#888] [General]
Hi,

I am working in a project using syncua, I am facing a bug that you have solved in master, but it has not been released yet. 
Do you have any idea when you are going to release a new version with all the improvements and bugfixes you have in master?

Thanks and regards

**Comment by oroulet** [comment]
I can release now, there are many fixes

**Reply by oroulet** [reply]
done. let me know if there are any issues

**Reply by developera** [reply]
New release working fine.
Thank you very much
